lib/tarantool_server.py

```diff
@@ -583,7 +583,7 @@ def script(self, val):
                 delattr(self, '_script')
             return
         self._script = os.path.abspath(val)
-        self.name = os.path.basename(self._script).split('.')[0]
+        self.name = re.sub(r".lua", "", os.path.basename(self._script))
```

Comment:
Imagine if the script is named `box.luafoobar.lua`. In this case, `self.name` will have an incorrect value.

Reply:
No, if the script is named `box.luafoobar.lua`, the `self.name` will have an expected value `box.luafoobar`